templates/overrides/dataset/table.html.twig

```diff
@@ -1,13 +0,0 @@
-{#
```

Comment:
This template can not be deleted. Must exists I suppose because dataset needs it and there is not one from system templates.

Reply:
That's weird. There definitely is a `table.html.twig` in system module.
So what happens if we delete this one?

Reply:
Yes, it was cache problem. Checked again and don't need to recover the pattern.